wp-content/themes/engage/assets/scss/components/_widget.scss

```diff
@@ -29,34 +29,59 @@
 		color: $black-grey;
 	}
 
-	.constantcontactwidget_form {
-		p {
-			margin-bottom: 1rem;
-			color: $black-grey;
-		}
+	.textwidget {
+		.wpcf7-form {
+			margin: 0;
 
-		input[type="text"] {
-			font: inherit;
-			margin-top: 1rem;
-			width: 100%;
+			.newsletter-prompt {
+				margin-bottom: 1.6rem;
+				color: $black-grey;
+			}
 
-			@include media($tablet) {
+			.newsletter-input {
+				font: inherit;
 				width: 100%;
+				border: 1px solid;
+
+				@include media($tablet) {
+					width: 100%;
+					border: none;
+				}
 			}
-		}
 
-		input[type="submit"] {
-			display: block;
-			background: $orange;
-			margin-top: 2.5rem;
-			margin-bottom: 1rem;
+			.newsletter-submit {
+				display: block;
+				width: 110px;
```

Comment:
Rather than add a specific width, add padding to the right/left to get the effect you want. Otherwise if the text ever becomes large it can't resize (since it's fixed at 110px).